src/components/historikk/Historikk.tsx

```diff
@@ -87,6 +71,37 @@ const Historikk = ({
     historikkEvents
   );
 
+  const UtvidbarHistorikk = (
+    props: EkspanderbartpanelBaseProps
+  ): JSX.Element => {
+    const StyledEkspanderbartpanel = styled(Ekspanderbartpanel)`
```

Comment:
Har vi en konvensjon for hva vi kaller elementer når de blir stylet? Hadde vært diggest å bare kalt denne `Ekspanderbartpanel`, men da blir det fort name conflict. Synes bare det ser litt funky ut når mye ender opp med å få en `Styled…` prefix🤔